schematics/types/compound.py

```diff
@@ -244,6 +244,13 @@ def _export(self, list_instance, format, context):
 class DictType(CompoundType):
     """A field for storing a mapping of items, the values of which must conform to the type
     specified by the ``field`` parameter.
+    
+    Use it like this:
+    
+    .. code-block:: python
```

Comment:
The `code-block` directive is unnecessary. A simple `::` marker is preferred (`Use it like this::`).